site/_docs/windows.md

```diff
@@ -44,3 +46,74 @@ with Windows. Add the following to the Gemfile for your site:
 {% highlight ruby %}
 gem 'wdm', '~> 0.1.0' if Gem.win_platform?
 {% endhighlight %}
+
+### How to install github-gem
+
+This section is part of an article written by [Jens Willmer][jwillmerPost]. To follow the instructions you need to have [Chocolatey][] installed on your system. If you already have a version of Ruby installed you need to uninstall it before you can continue.
+
+#### Install Ruby and Ruby development kit
+
+Open a command prompt and execute the following commands:
+
+ * `choco install ruby -version 2.2.4` 
+ * `choco install ruby2.devkit` - _needed for compilation of json gem_
+ 
+#### Configure Ruby development kit 
+
+The development kit did not set the environment path for Ruby so we need to do it.
+ 
+ * Open command prompt in `C:\tools\DevKit2` 
+ * Execute `ruby dk.rb init` to create a file called `config.yml`
+ * Edit the `config.yml` file and include the path to Ruby `- C:/tools/ruby22`
+ * Execute the following command to set the path: `ruby dk.rb install`
+
+#### Nokogiri gem installation
+
+This gem is also needed in the github-gem and to get it running on Windows x64 we have to install a few things.
+
+
+**Note:** In the current [pre release][nokogiriFails] it works out of the box with Windows x64 but this version is not referenced in the github-gem.
+
+
+`cinst -Source "https://go.microsoft.com/fwlink/?LinkID=230477" libxml2`{:.language-ruby}
+
+`cinst -Source "https://go.microsoft.com/fwlink/?LinkID=230477" libxslt`{:.language-ruby}
+
+`cinst -Source "https://go.microsoft.com/fwlink/?LinkID=230477" libiconv`{:.language-ruby}
+ 
+```language-ruby
+ gem install nokogiri --^
+   --with-xml2-include=C:\Chocolatey\lib\libxml2.2.7.8.7\build\native\include^
+   --with-xml2-lib=C:\Chocolatey\lib\libxml2.redist.2.7.8.7\build\native\bin\v110\x64\Release\dynamic\cdecl^
+   --with-iconv-include=C:\Chocolatey\lib\libiconv.1.14.0.11\build\native\include^
+   --with-iconv-lib=C:\Chocolatey\lib\libiconv.redist.1.14.0.11\build\native\bin\v110\x64\Release\dynamic\cdecl^
+   --with-xslt-include=C:\Chocolatey\lib\libxslt.1.1.28.0\build\native\include^
+   --with-xslt-lib=C:\Chocolatey\lib\libxslt.redist.1.1.28.0\build\native\bin\v110\x64\Release\dynamic
+```
+
+#### Install github-gem
```

Comment:
The gem is technically called `github-pages` – could you update all your references to use this consistent name?